lib/Doctrine/ORM/Mapping/Driver/DatabaseDriver.php

```diff
@@ -168,6 +175,16 @@ public function setTables($entityTables, $manyToManyTables)
         }
     }
 
+    public function setInflector(Inflector $inflector) : void
+    {
+        $this->inflector = $inflector;
+    }
+
+    public function getInflector() : Inflector
+    {
+        return $this->Inflector;
```

Comment:
Inflector is badly inflected here :P
```suggestion
        return $this->inflector;
```